tests/test_common.py

```diff
@@ -91,13 +91,17 @@ def __init__(self):
             name="foo",
             version="bar",
             kinematics=CenterOfMass(10, "p", "p"),
+            _frame=EventFrame.CENTER_OF_MASS,
         )
 
         super().__init__(generator)
 
     def _charge_init(self, npart):
         return np.zeros(npart)
 
+    def _add_init_beam_info(self):
+        self._append_initial_beam()
```

Comment:
There is no test that checks that you have successfully added the particles in the correct positon in the event.

Reply:
Ok. The `test_models_beam` is added. It tests beam values with first 2 entries of event.